deps/media-playback/CMakeLists.txt

```diff
@@ -28,6 +28,9 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(powerpc|ppc)64le")
 			PUBLIC
 			-mvsx)
 	add_compile_definitions(NO_WARN_X86_INTRINSICS)
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
```

Comment:
Just make the check below appropriate instead of adding this which does nothing and depends on the order of the if statements.

Reply:
This is absolutely necessary, as otherwise the `elseif(NOT MSVC)` immediately following applies the -mmmx, -msse, and -msse2 flags which break portability. (Also, depending on the compiler these may not even be necessary)

Now if you are asking me to clean up the entire selection process, that's fine too, but please be specific.

Reply:
indeed the "check below" im referring to is the MSVC check. As I see it we shouldnt blindly apply these extensions if there is already an architecture check available to us. This also makes support aarch64 simpler since it will do nothing.